types/activity/submission.ts

```diff
@@ -19,7 +19,7 @@ export type SubmissionCreatedActivityItem = SubmissionActivityItemBase & {
 };
 
 export type SubmissionUpdatedActivityItem = SubmissionActivityItemBase & {
-	kind: 'submission-updated';
+	kind: 'submission-status-updated';
 	payload: {
 		status?: Diff<Submission['status']>;
```

Comment:
Since this item is now exclusively used for status updates, we can make the `status` in the payload non-optional:

```ts
status: Diff<Submission['status']>;
```

This allows us to remove the `if (payload.status)` check in the renderer, and also to get rid of the `message !== null` check in `renderActivityItem`.